javascript/ql/src/semmle/javascript/security/dataflow/IndirectCommandInjection.qll

```diff
@@ -0,0 +1,83 @@
+/**
+ * Provides a taint-tracking configuration for reasoning about command-injection
+ * vulnerabilities (CWE-078).
+ */
+
+import javascript
+import IndirectCommandArgument
+
+module IndirectCommandInjection {
+  /**
+   * A data-flow source for command-injection vulnerabilities.
+   */
+  abstract class Source extends DataFlow::Node { }
```

Comment:
Perhaps it would make sense to pull out `Source` and `Sink` like we did for the other configurations?